.travis.yml

```diff
@@ -11,14 +11,16 @@ addons:
     packages:
     - lua5.1
     - libxml2-utils
-    - python-yaml
-    - postgresql-9.4-postgis-2.3
+    - python3-yaml
+    - postgresql-9.4-postgis-2.4
```

Comment:
Does this package exist? In bionic there seems to be `postgresql-10-postgis-2.4`.